knowledge-base/form-focus-formitem.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Set Programmatic Focus to a FormItem
```

Comment:
The customer has used the keyword "initial", which is not present anywhere in this article.